Source/relay/Guide/Sections/Messages.ash

```diff
@@ -573,7 +573,7 @@ string generateRandomMessage()
     else if (current_hour == 6)
         random_messages.listAppend("the dawn is your enemy");
     
-    random_messages.listAppend("take chances, be courages, go exploring!");
```

Comment:
I'm pretty sure it was a reference to something... eh, whatever

Reply:
Could be...maybe I just don't know it.